CMakeLists.txt

```diff
@@ -505,6 +508,15 @@ if(BUILD_TESTING)
   rcutils_custom_add_gtest(test_macros
     test/test_macros.cpp
   )
+
+  add_performance_test(benchmark_logging test/benchmark/benchmark_logging.cpp)
+  target_link_libraries(benchmark_logging ${PROJECT_NAME})
+  target_include_directories(benchmark_logging PUBLIC include)
```

Comment:
These should get the same `if(TARGET ...)` treatment that the gtests are getting above. If `benchmark` isn't available, the tests won't be built, and these lines will fail (the same as gtest).

Reply:
Sure, see b830dba